.circleci/config.yml

```diff
@@ -86,16 +54,20 @@ workflows:
             tags:
               only: /^v.*/
 
-      - architect/push-to-docker-legacy:
-          name: push-chart-operator-to-aliyun-legacy
-          image: "registry-intl.cn-shanghai.aliyuncs.com/giantswarm/chart-operator"
-          username_envar: "ALIYUN_USERNAME"
-          password_envar: "ALIYUN_PASSWORD"
+      - architect/push-to-app-catalog:
+          name: push-chart-operator-to-control-plane-app-catalog
+          app_catalog: "control-plane-catalog"
+          app_catalog_test: "control-plane-test-catalog"
+          chart: "chart-operator"
           requires:
-            - build
+            - push-chart-operator-to-quay
```

Comment:
Recently we decided to wait for aliyun too.

Reply:
Ack. Done.